src/components/Invoice/EditInvoice.tsx

```diff
@@ -0,0 +1,15 @@
+import { useParams } from 'react-router-dom';
+import InvoiceForm from './InvoiceForm';
+import { useInvoice } from './use-invoice';
+import { InvoiceParamsSchema } from './params.schema';
```

Comment:
Every other schema is named with lowercase, only this one is Capitalised. I'd suggest making it consistent :)

Reply:
done